src/test/java/org/opentripplanner/standalone/config/RouterConfigTest.java

```diff
@@ -20,17 +20,17 @@ class RouterConfigTest {
    */
   @Test
   void validateBuildConfigExample() {
```

Comment:
Not really part of this pr, but the name could be fixed
```suggestion
  void validateRouterConfigExample() {
```

Reply:
I fixed this - the fix will be in Part 6.